CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 * [CHANGE] StoreGateway/Alertmanager: Add default 5s connection timeout on client. #6603
+* [FEATURE] Query Frontend: Add new limit `-frontend.max-query-response-size` for total query response size after decompression in query frontend. #6607
```

Comment:
Can we change this to `ENHANCEMENT`? Since it is more a limit to protect the service rather than a new feature